hayeon/programmers-0/직사각형-넓이-구하기.js

```diff
@@ -0,0 +1,16 @@
+function solution(dots) {
+  //const width = Math.abs(dots[0][0] - dots[1][0]);
+  //const height = Math.abs(dots[2][1] - dots[1][1]);
+
+  const width = [];
+  const height = [];
+
+  for (let el of dots) {
+    width.push(el[0]);
+    height.push(el[1]);
+  }
+  return (
+    (Math.max(...width) - Math.min(...width)) *
+    (Math.max(...height) - Math.min(...height))
+  );
+}
```

Comment:
```suggestion
function solution(dots) {
  const width = dots.map(el => el[0]);
  const height = dots.map(el => el[1]);

  return (
    (Math.max(...width) - Math.min(...width)) *
    (Math.max(...height) - Math.min(...height))
  );
}
```
- 풀이 잘봤습니다! for문을 통해 하나하나 push하는 것 보다 map을 이용해 도출하는 방법도 있습니다 참고해주세요 👍